jupyterlab_widgets/pyproject.toml

```diff
@@ -1,3 +1,3 @@
 [build-system]
-requires = ["jupyter_packaging~=0.7.9", "jupyterlab~=3.0", "setuptools>=40.8.0", "wheel"]
```

Comment:
Why these changes? I believe that the pin on `jupyter_packaging` is important as the API for that package is subject to change.

Reply:
The build was broken before these changes (see commit 2b9a2d74). I didn't want to pin a version of jupyter_packaging but 7.9 doesn't have get_version_info, so I just left it out. I can add a TODO to figure out what version of jupyter_packaging to use given that it's out of scope for this PR. (I think get_version_info was introduced with 9.0.)

Reply:
Maybe we could then update to jupyter-packaging 0.9 in a separate PR so it's easier to track?

Reply:
Opened https://github.com/jupyter-widgets/ipywidgets/pull/3194

Reply:
Thanks!